Specs/Data/Cesium3DTiles/Vector/tileset.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
This tileset doesn't seem to render for me. Also for future proofing all these should be placed in their own subfolder in Vector/ since we will likely add more tilesets as we go.